dist/angular-google-analytics.js

```diff
@@ -696,12 +696,25 @@
             if (angular.isDefined(noninteraction)) {
               opt_fieldObject.nonInteraction = !!noninteraction;
             }
+            
+            if (angular.isDefined(transport) && angular.isDefined(href)) {
+                opt_fieldObject.transport = transport;
+                opt_fieldObject.hitCallback = loadPage;
+            } else if (angular.isDefined(transport) || angular.isDefined(href)) {
+                alert("transport and outbound href required")
```

Comment:
Please make this more verbose and do not use an alert. We are using the $log service, this being a $log.warning at best, as it is testable.